src/scss/tools/_index.scss

```
@@ -1,12 +1,12 @@
 // =================================
-// Tools
+// Fozzie SCSS Utilities
```

I'd keep this as tools, as it's the index of that folder

```suggestion
// Tools
```